test/libsolidity/syntaxTests/tupleAssignments/empty_tuples_lhs.sol

```diff
@@ -26,4 +26,4 @@ contract C {
 // TypeError 5547: (401-404): Empty tuple on the left hand side.
```

Comment:
More syntax test ideas:
- Error name qualified by module/contract name.
- Error declaration with parameters marked as `indexed`.
- Error declaration with parameters marked as `memory/calldata/storage`.
- Catching an error.
- Error declared in a contract shadowing a globally declared error.
- Error declaration in an interface or in a library.
- Trying to get a function pointer to a error.
- Errors in constructor or functions assigned to storage variables.
- Error type as a function parameter.
- Error type with contract as a parameter.
- Passing errors to `abi.encode()`/`abi.decode()`.

Reply:
Added tests for most of these.